orchestra/interface_api/project_management/views.py

```diff
@@ -34,9 +38,20 @@ class ProjectList(generics.ListCreateAPIView):
     queryset = Project.objects.exclude(status=Project.Status.ABORTED)
 
 
-@project_management_api_view
+@project_management_api_view_base
 def project_information_api(request):
+    """
+    This function is used by both the project management interface
+    (project admins only) and for providing project information to
+    experts (only to experts associated with a project). We enforce
+    both of these permissions in the view below.
+
+    """
     project_id = load_encoded_json(request.body)['project_id']
+    worker = get_object_or_404(Worker, user=request.user)
+    if not (is_project_admin(request.user) or
+            worker.assignments.filter(task__project=project_id).exists()):
+        raise BadRequest('Permission denied')
```

Comment:
should be a 403 or 404 on Permission denied, not a 400?

Reply:
Also looks like you created nice permission classes for this in your PR, can they be reused here?

Reply:
There's only one exception type for jsonviews: https://github.com/jsocol/django-jsonview/blob/master/jsonview/exceptions.py#L1

Reply:
`permission_classes` by default can only be ANDed, not ORed. [This project](https://github.com/niwinz/djangorestframework-composed-permissions) addresses that, and we can consider depending on it in a followon PR

Reply:
just to win an argument, [just return Django's `PermissionDenied`](https://github.com/jsocol/django-jsonview#return-values)

Reply:
ok